components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -253,4 +278,26 @@ std::string ViewCounterService::GetSuperReferralCode() const {
   return service_->GetSuperReferralCode();
 }
 
+void ViewCounterService::UpdateP3AValues() const {
+  uint64_t new_tab_count = new_tab_count_state_->GetHighestValueInWeek();
+  constexpr int kNewTabCount[] = {0, 1, 4, 9, 21, 51, 101};
+  const int* it_count =
+      std::lower_bound(kNewTabCount, std::end(kNewTabCount), new_tab_count);
+  int answer = it_count - kNewTabCount;
+  UMA_HISTOGRAM_EXACT_LINEAR("Brave.NTP.NewTabsCreated", answer, 6);
```

Comment:
I think it should not 6, but std::size(kNewTabCount) + 1, because it looks like 8 buckets: 7 for values in the array and one "overflow"